src/Components/SmartComponents/SystemCves/SystemCveTableToolbar.js

```diff
@@ -56,8 +56,10 @@ const SystemCveToolbarWithContext = ({
         multiRow: true
     }), [selectedCves, cves, systemCVEs, parameters, methods]);
 
+    const canSelect = envContext.isPairStatusEditable || envContext.isRemediationEnabled;
+
     const actions = ['',
-        ...canEditPairStatus ?
+        ...envContext.isPairStatusEditable ?
```

Comment:
I would just spread all the needed functions and flags right in the useContext hook call

Reply:
I was thinking considering it, but I find this way to be more readable since you can at a glance distinguish that this feature toggle is related to `envContext`.